multi/stm32l4-multi/stm32-multi.c

```diff
@@ -163,7 +163,7 @@ static void handleMsg(msg_t *msg)
 
 static ssize_t console_write(const char *str, size_t len, int mode)
 {
-#if TTY1 || TTY2 || TTY3 || TTY4 || TTY5
+#if TTY_CONSOLE
```

Comment:
Where is `TTY_CONSOLE` defined?

Reply:
It;s not. I missed this during rewrite. This is CONSOLE_IS_TTY now.